frontend/www/js/omegaup/components/common/LogoutConfirmation.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <b-modal
+    v-model="showModal"
+    :title="T.logoutConfirmationTitle"
+    :ok-title="T.wordsYes"
+    :no-title="T.wordsNo"
```

Comment:
I realized something: the button that should display "No" was displaying "Cancel" in all three languages. So, I checked the documentation and noticed that the attribute has a different name:
```suggestion
    :cancel-title="T.wordsNo"
```
With this change, it should now display the correct text.

Reply:
This suggestion still needs to be addressed as well